tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -90,10 +100,13 @@ public WorkflowRunner(String entry, String version, String pathOfTestParameter,
      * @param ga4Ghv20Api
      * @param extendedGa4GhApi
      */
-    public WorkflowRunner(String entry, String version, String relativePathToTestParameterFile, Ga4Ghv20Api ga4Ghv20Api, ExtendedGa4GhApi extendedGa4GhApi)  {
+    @SuppressWarnings("checkstyle:parameternumber")
+    public WorkflowRunner(String entry, String version, String relativePathToTestParameterFile, Ga4Ghv20Api ga4Ghv20Api, ExtendedGa4GhApi extendedGa4GhApi, WorkflowsApi workflowsApi, String WDLConfigFilePath, String CWLConfigFilePath)  {
```

Comment:
the github code highlighting is a bit messed here, this isn't a typical naming pattern for parameters (leading caps)

Reply:
also, this constructor is very similar to the first constructor, you can probably chain constructors to remove the repetitiveness https://www.scientecheasy.com/2020/06/java-constructor-chaining.html/

Reply:
Fixed

Reply:
parameter naming is still non-standard, constructor is better though

Reply:
Sorry, completely missed the bit about the naming pattern.

It is now fixed.